book/src/dev/rfcs/xxxx-block-subsidy.md

```diff
@@ -47,121 +58,130 @@ In addition to calculations the block subsidy requires constants defined in the
-- `PRE_BLOSSOM_HALVING_INTERVAL`
-- `POST_BLOSSOM_HALVING_INTERVAL`
-- `FOUNDERS_FRACTION_DIVISOR`
-- `FOUNDER_ADDRESS_CHANGE_INTERVAL`
-- `FOUNDERS_REWARD_ADDRESSES_MAINNET`
-- `FOUNDERS_REWARD_ADDRESSES_TESTNET`
+- `SLOW_START_INTERVAL: Height`
+- `SLOW_START_SHIFT: Height`
+- `MAX_BLOCK_SUBSIDY: u64`
+- `PRE_BLOSSOM_POW_TARGET_SPACING: Duration`
+- `POST_BLOSSOM_POW_TARGET_SPACING: Duration`
+- `BLOSSOM_POW_TARGET_SPACING_RATIO: u64`
+- `PRE_BLOSSOM_HALVING_INTERVAL: Height`
+- `POST_BLOSSOM_HALVING_INTERVAL: Height`
+- `FOUNDERS_FRACTION_DIVISOR: u64`
+- `FOUNDER_ADDRESS_CHANGE_INTERVAL: u64`
+- `FOUNDER_ADDRESSES_MAINNET: [&'static str; 48]`
+- `FOUNDER_ADDRESSES_TESTNET: [&'static str; 48]`
 
 ### Funding streams parameter constants
 
 The design suggests to implement the parameters needed for funding streams as:
 
 ```
 /// The funding stream receiver categories
-pub enum FundingStreamsReceivers {
-    /// Electric Coin Company
-    ECC,
-    /// ZCash Foundation
-    ZF,
-    /// Major Grants
-    MG,
+#[allow(missing_docs)]
+pub enum FundingStreamReceiver {
+    ElectricCoinCompany,
+    ZcashFoundation,
+    MajorGrants,
 }
 
 /// The numerator for each funding stream receiving category 
 /// as described in [protocol specification §7.9.1][7.9.1].
 /// 
 /// [7.9.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
-const FUNDINGSTREAMS_RECEIVERS_NUMERATORS: &[(i32, FundingStreamsReceivers)] = &[
-    (7, FundingStreamsReceivers::ECC),
-    (5, FundingStreamsReceivers::ZF),
-    (8, FundingStreamsReceivers::MG),
+const FUNDING_STREAM_RECEIVER_NUMERATORS: &[(FundingStreamReceiver, u64)] = &[
+    (FundingStreamReceiver::ElectricCoinCompany, 7),
+    (FundingStreamReceiver::ZcashFoundation, 5),
+    (FundingStreamReceiver::MajorGrants, 8),
 ];
 
 /// Denominator as described in [protocol specification §7.9.1][7.9.1].
 /// 
 /// [7.9.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
-pub const FUNDINGSTREAM_DENOMINATOR: i32 = 100;
-
-/// Height intervals for funding streams on each network.
-pub enum FundingStreamsHeightIntervals {
-    /// For the Mainnet
-    Mainnet,
-    /// For the Testnet
-    Testnet,
+pub const FUNDING_STREAM_RECEIVER_DENOMINATOR: u64 = 100;
+
+#[allow(missing_docs)]
+pub enum FundingStreamRange {
+    StartHeight,
+    EndHeight,
 }
 
 /// Start and end Heights for funding streams 
 /// as described in [protocol specification §7.9.1][7.9.1].
 /// 
 /// [7.9.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
-const FUNDINGSTREAMS_HEIGHT_INTERVALS: &[(Height, Height, FundingStreamsHeightIntervals)] = &[
-    (Height(1_046_400), Height(2_726_400), FundingStreamsHeightIntervals::Mainnet),
-    (Height(1_028_500), Height(2_796_000), FundingStreamsHeightIntervals::Testnet),
+const FUNDING_STREAM_HEIGHT_RANGES: &[(Network, Height, Height)] = &[
+    (Network::Mainnet, Height(1_046_400), Height(2_726_400)),
+    (Network::Testnet, Height(1_028_500), Height(2_796_000)),
 ];
```

Comment:
This range includes the first block, but excludes the last block, just like the Rust `Range` type:
```suggestion
const FUNDING_STREAM_HEIGHT_RANGES: &[(Network, Range)] = &[
    (Network::Mainnet, Height(1_046_400)..Height(2_726_400)),
    (Network::Testnet, Height(1_028_500)..Height(2_796_000)),
];
```

Reply:
Nice one, added at https://github.com/ZcashFoundation/zebra/pull/1129/commits/589cef37e1886e898b74ff153ac3dd38e4bce86d